src/command_modules/azure-cli-sql/tests/test_sql_commands.py

```diff
@@ -290,6 +290,41 @@ def test_sql_db_mgmt(self, resource_group, resource_group_location, server):
                  checks=[NoneCheck()])
 
 
+class AzureActiveDirectoryAdministratorScenarioTest(ScenarioTest):
+    @ResourceGroupPreparer()
+    @SqlServerPreparer()
+    def test_aad_admin(self, resource_group, server):
+        rg = resource_group
+        sn = server
+        oid = '5e90ef3b-9b42-4777-819b-25c36961ea4d'
+        tid = '72f988bf-86f1-41af-91ab-2d7cd011db47'
+        user = 'DSEngAll'
+
+        self.cmd('sql server ad-admin create -s {} -g {} -i {} -u {}'
+                 .format(sn, rg, oid, user),
+                 checks=[JMESPathCheck('login', user),
+                         JMESPathCheck('sid', oid),
+                         JMESPathCheck('tenantId', tid)])
+
+        self.cmd('sql server ad-admin list -s {} -g {}'
+                 .format(sn, rg),
+                 checks=[JMESPathCheck('[0].login', user)])
+
+        user2 = 'TestUser'
+
+        self.cmd('sql server ad-admin update -s {} -g {} -u {} -i {}'
+                 .format(sn, rg, user2, oid),
```

Comment:
update to a *different* oid so that you are testing that update actually uses the specified oid